### PR TITLE
#26 infinite mints

### PR DESCRIPTION
**Pull request by Mario-SO**
There is now a toggle that lets users decide wether they want a limited amount of people to be able to subscribe, or let an infinite amount of people subscribe.

### CODE REVIEW

File: client/components/Router.tsx [hunk 1]
```diff
@@ -9,7 +9,7 @@ const Layout = ({ children }: { children: React.ReactNode }) => (
   <div className=" h-screen mx-auto w-screen">{children}</div>
 );
 const contractAddress = process.env.NEXT_PUBLIC_FACTORY_CONTRACT_ADDRESS ?? "";
-const protectedRoutes = ["/", "/create", "/list", "/golive"];
+const protectedRoutes = ["/", "/list", "/golive"];
```
Comment (pabloVoorvaart) [inline client/components/Router.tsx:12]: add "/create"

File: client/hooks/useCreateLiveNFT.tsx [hunk 2]
```diff
@@ -39,12 +39,13 @@ const useCreateLiveNFT = () => {
       functionName: "createLiveNFT",
       args: validateFormData()
         ? [
```
Comment (pabloVoorvaart) [inline client/hooks/useCreateLiveNFT.tsx:41]: set order as it is in init function

File: client/pages/create.tsx [hunk 3]
```diff
@@ -10,7 +10,7 @@ import GoBackButton from "components/Buttons/GoBackButton";
 
 
 const CreateComponent = () => {
-  const [step, setStep] = React.useState(0);
+  const [step, setStep] = React.useState(2);
```
Comment (pabloVoorvaart) [inline client/pages/create.tsx:13]: set should be 0

**Review by pabloVoorvaart — CHANGES_REQUESTED** [review]

**Review by pabloVoorvaart — APPROVED** [review]
lgtm!

File: client/hooks/useCreateLiveNFT.tsx (hunk 2 above)
Comment (Mario-SO) [reply client/hooks/useCreateLiveNFT.tsx:41]: This was already in the correct order

**Pull request merged**